cherryontech/jupiter-jumpers
# Summary + Filter

## Description

Designed to display summarized mood data. 

## Related Issue

https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-54


## Type of Changes

Created the summary of moods for user.  User is also able to filter through moods according to "rainy, story, cloudy..." 

## Acceptance Criteria

- [ ] The summary page component is successfully created.
- [ ] Dates, moods, and mood names are rendered dynamically within the component.
- [ ] User is able to filter through the summary depending on the mood. 

## Update Screenshots
<img width="1473" alt="Screenshot 2024-05-15 at 9 38 49 PM" src="https://github.com/cherryontech/jupiter-jumpers/assets/123599985/66f89785-33b3-41e4-af12-9be38b7e5953">


## Testing Instructions

1. While logged in as a user, make sure you can see past logs in the summary section (otherwise blank). 
2. Once daily logged is saved, it will appear on the summary page.  
3. Check that dates, moods, and mood names are displayed accurately.
4. Try the drop down filter to filter per mood given for each month.
